CleanWeather/Scenes/CityListMVVM/CityListCellManagerMVVM.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  CityListCellManagerMVVM.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 11/01/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import UIKit
+
+final class CityListCellManagerMVVM: NSObject {
+
+    var citiesWeather = [CityWeather]()
+    private var citiesWeatherDisplayable: [CityWeatherDisplayable] {
+        return citiesWeather.map { CityWeatherDisplayable(object: $0)}
+    }
+}
+
+extension CityListCellManagerMVVM {
+    
+    func buildCell(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeue(with: CityListTableViewCell.self, for: indexPath)
+        cell.setup(with: citiesWeatherDisplayable[indexPath.row])
+        return cell
+    }
+
+    func didSelectCityCell(indexPath: IndexPath) -> CityWeather {
```

Comment:
you can keep it only in controller, no need to continue asking for cell manager in controllers tableview delegate